2021-06-01 14:55:10.093  WARN 37367 --- [pool-1-thread-4] com.zaxxer.hikari.pool.ProxyConnection   : HikariPool-1 - Connection org.postgresql.jdbc.PgConnection@413555e7 marked as broken because of SQLSTATE(08003), ErrorCode(0)

org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:877) ~[postgresql-42.2.19.jar:42.2.19]
	at org.postgresql.jdbc.PgConnection.rollback(PgConnection.java:884) ~[postgresql-42.2.19.jar:42.2.19]
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396) ~[HikariCP-3.4.5.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java) ~[HikariCP-3.4.5.jar:na]
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:263) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:101) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:562) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:743) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.3.6.jar:5.3.6]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:832) ~[na:na]

2021-06-01 14:55:10.105 ERROR 37367 --- [pool-1-thread-4] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void com.josh.emailFunctionality.service.EmailSendServiceImpl.sendEmail(com.josh.emailFunctionality.entity.EmailEntity) throws java.lang.Exception

org.springframework.orm.jpa.JpaSystemException: Unable to rollback against JDBC Connection; nested exception is org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:331) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:233) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:566) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:743) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.3.6.jar:5.3.6]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:832) ~[na:na]
Caused by: org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:127) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:263) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:101) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:562) ~[spring-orm-5.3.6.jar:5.3.6]
	... 13 common frames omitted
Caused by: org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:877) ~[postgresql-42.2.19.jar:42.2.19]
	at org.postgresql.jdbc.PgConnection.rollback(PgConnection.java:884) ~[postgresql-42.2.19.jar:42.2.19]
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396) ~[HikariCP-3.4.5.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java) ~[HikariCP-3.4.5.jar:na]
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	... 17 common frames omitted

2021-06-01 14:55:11.539  WARN 37367 --- [pool-1-thread-2] com.zaxxer.hikari.pool.ProxyConnection   : HikariPool-1 - Connection org.postgresql.jdbc.PgConnection@3238b0a7 marked as broken because of SQLSTATE(08003), ErrorCode(0)

org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:877) ~[postgresql-42.2.19.jar:42.2.19]
	at org.postgresql.jdbc.PgConnection.rollback(PgConnection.java:884) ~[postgresql-42.2.19.jar:42.2.19]
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396) ~[HikariCP-3.4.5.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java) ~[HikariCP-3.4.5.jar:na]
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:263) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:101) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:562) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:743) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.3.6.jar:5.3.6]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:832) ~[na:na]

2021-06-01 14:55:11.548 ERROR 37367 --- [pool-1-thread-2] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void com.josh.emailFunctionality.service.EmailSendServiceImpl.sendEmail(com.josh.emailFunctionality.entity.EmailEntity) throws java.lang.Exception

org.springframework.orm.jpa.JpaSystemException: Unable to rollback against JDBC Connection; nested exception is org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:331) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:233) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:566) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:743) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.3.6.jar:5.3.6]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:832) ~[na:na]
Caused by: org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:127) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:263) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:101) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:562) ~[spring-orm-5.3.6.jar:5.3.6]
	... 13 common frames omitted
Caused by: org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:877) ~[postgresql-42.2.19.jar:42.2.19]
	at org.postgresql.jdbc.PgConnection.rollback(PgConnection.java:884) ~[postgresql-42.2.19.jar:42.2.19]
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396) ~[HikariCP-3.4.5.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java) ~[HikariCP-3.4.5.jar:na]
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	... 17 common frames omitted

2021-06-01 14:55:12.248  WARN 37367 --- [pool-1-thread-1] com.zaxxer.hikari.pool.ProxyConnection   : HikariPool-1 - Connection org.postgresql.jdbc.PgConnection@76b4caa5 marked as broken because of SQLSTATE(08003), ErrorCode(0)

org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:877) ~[postgresql-42.2.19.jar:42.2.19]
	at org.postgresql.jdbc.PgConnection.rollback(PgConnection.java:884) ~[postgresql-42.2.19.jar:42.2.19]
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396) ~[HikariCP-3.4.5.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java) ~[HikariCP-3.4.5.jar:na]
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:263) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:101) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:562) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:743) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.3.6.jar:5.3.6]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:832) ~[na:na]

2021-06-01 14:55:12.253 ERROR 37367 --- [pool-1-thread-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void com.josh.emailFunctionality.service.EmailSendServiceImpl.sendEmail(com.josh.emailFunctionality.entity.EmailEntity) throws java.lang.Exception

org.springframework.orm.jpa.JpaSystemException: Unable to rollback against JDBC Connection; nested exception is org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:331) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:233) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:566) ~[spring-orm-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:743) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:711) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:654) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:407) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750) ~[spring-aop-5.3.6.jar:5.3.6]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.3.6.jar:5.3.6]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:832) ~[na:na]
Caused by: org.hibernate.TransactionException: Unable to rollback against JDBC Connection
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:127) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.rollback(JdbcResourceLocalTransactionCoordinatorImpl.java:304) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:263) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:101) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:562) ~[spring-orm-5.3.6.jar:5.3.6]
	... 13 common frames omitted
Caused by: org.postgresql.util.PSQLException: This connection has been closed.
	at org.postgresql.jdbc.PgConnection.checkClosed(PgConnection.java:877) ~[postgresql-42.2.19.jar:42.2.19]
	at org.postgresql.jdbc.PgConnection.rollback(PgConnection.java:884) ~[postgresql-42.2.19.jar:42.2.19]
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:396) ~[HikariCP-3.4.5.jar:na]
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java) ~[HikariCP-3.4.5.jar:na]
	at org.hibernate.resource.jdbc.internal.AbstractLogicalConnectionImplementor.rollback(AbstractLogicalConnectionImplementor.java:121) ~[hibernate-core-5.4.30.Final.jar:5.4.30.Final]
	... 17 common frames omitted

2021-06-01 14:55:36.372  INFO 37367 --- [RMI TCP Connection(26)-127.0.0.1] inMXBeanRegistrar$SpringApplicationAdmin : Application shutdown requested.
2021-06-01 14:55:36.388  INFO 37367 --- [RMI TCP Connection(26)-127.0.0.1] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2021-06-01 14:55:36.389  INFO 37367 --- [RMI TCP Connection(26)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Shutdown initiated...
2021-06-01 14:55:36.409  INFO 37367 --- [RMI TCP Connection(26)-127.0.0.1] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Shutdown completed.
2021-06-01 15:14:56.951  INFO 39132 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39132 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:14:56.968  INFO 39132 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:14:58.463  INFO 39132 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 2.142 seconds (JVM running for 4.094)
2021-06-01 15:14:58.704  INFO 39132 --- [smtp:127.0.0.1:3025<-/127.0.0.1:46069] com.icegreen.greenmail.smtp.SmtpManager  : Created user login test@gmail.com for address test@gmail.com with password test@gmail.com because it didn't exist before.
2021-06-01 15:14:58.725 ERROR 39132 --- [main] c.j.e.service.EmailSendServiceImpl       :  In exception of AuthenticationFailedException 
2021-06-01 15:14:58.729 ERROR 39132 --- [main] c.j.e.service.EmailSendServiceImpl       : In exception of SMTPSendFailedException daily limit failed for test@gmail.com
2021-06-01 15:14:58.740 ERROR 39132 --- [main] c.j.e.service.EmailSendServiceImpl       : In exception of MailConnectException
2021-06-01 15:14:58.743 ERROR 39132 --- [main] c.j.e.service.EmailSendServiceImpl       : In exception of SMTPAddressFailedException
2021-06-01 15:14:58.749 ERROR 39132 --- [main] c.j.e.service.EmailSendServiceImpl       : Other exception For final common exception
2021-06-01 15:14:58.861  INFO 39132 --- [main] .b.t.c.SpringBootTestContextBootstrapper : Neither @ContextConfiguration nor @ContextHierarchy found for test class [com.josh.emailfunctionality.service.EmailRegisterServiceImplTest], using SpringBootContextLoader
2021-06-01 15:14:58.912  INFO 39132 --- [main] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [com.josh.emailfunctionality.service.EmailRegisterServiceImplTest]: no resource found for suffixes {-context.xml, Context.groovy}.
2021-06-01 15:14:58.946  INFO 39132 --- [main] .b.t.c.SpringBootTestContextBootstrapper : Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
2021-06-01 15:14:58.952  INFO 39132 --- [main] .b.t.c.SpringBootTestContextBootstrapper : Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@679dd234, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@60cb1ed6, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@1e5eb20a, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@4538856f, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@4c3de38e, org.springframework.test.context.support.DirtiesContextTestExecutionListener@74b86971, org.springframework.test.context.transaction.TransactionalTestExecutionListener@1f9d4b0e, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@1e8fb66f, org.springframework.test.context.event.EventPublishingTestExecutionListener@3d8d17a3, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@ac91282, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@7f79edee, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@1ca610a0, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@49433c98, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@b5c6a30, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@3bfae028]
2021-06-01 15:14:59.080  INFO 39132 --- [main] c.j.e.s.EmailRegisterServiceImplTest     : Starting EmailRegisterServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39132 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:14:59.085  INFO 39132 --- [main] c.j.e.s.EmailRegisterServiceImplTest     : No active profile set, falling back to default profiles: default
2021-06-01 15:14:59.151  INFO 39132 --- [main] c.j.e.s.EmailRegisterServiceImplTest     : Started EmailRegisterServiceImplTest in 0.18 seconds (JVM running for 4.782)
2021-06-01 15:14:59.239  INFO 39132 --- [main] o.a.maven.surefire.booter.ForkedBooter   : Starting ForkedBooter v2.22.2 using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39132 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:14:59.240  INFO 39132 --- [main] o.a.maven.surefire.booter.ForkedBooter   : No active profile set, falling back to default profiles: default
2021-06-01 15:15:00.557  INFO 39132 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2021-06-01 15:15:00.651  INFO 39132 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 83 ms. Found 2 JPA repository interfaces.
2021-06-01 15:15:01.875  INFO 39132 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2021-06-01 15:15:01.894  INFO 39132 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2021-06-01 15:15:01.897  INFO 39132 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.45]
2021-06-01 15:15:02.066  INFO 39132 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2021-06-01 15:15:02.067  INFO 39132 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2795 ms
2021-06-01 15:15:02.359  INFO 39132 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2021-06-01 15:15:02.448  INFO 39132 --- [main] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.30.Final
2021-06-01 15:15:02.514  INFO 39132 --- [main] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.2.Final}
2021-06-01 15:15:02.728  INFO 39132 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2021-06-01 15:15:03.289  INFO 39132 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2021-06-01 15:15:03.307  INFO 39132 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL81Dialect
2021-06-01 15:15:04.103  INFO 39132 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2021-06-01 15:15:04.113  INFO 39132 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2021-06-01 15:15:05.344  WARN 39132 --- [main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2021-06-01 15:15:06.201  INFO 39132 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 35619 (http) with context path ''
2021-06-01 15:15:06.780  INFO 39132 --- [main] o.a.maven.surefire.booter.ForkedBooter   : Started ForkedBooter in 7.61 seconds (JVM running for 12.411)
2021-06-01 15:15:06.856  INFO 39132 --- [main] o.a.maven.surefire.booter.ForkedBooter   : Starting ForkedBooter v2.22.2 using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39132 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:15:06.856  INFO 39132 --- [main] o.a.maven.surefire.booter.ForkedBooter   : No active profile set, falling back to default profiles: default
2021-06-01 15:15:07.324  INFO 39132 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2021-06-01 15:15:07.337  INFO 39132 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 12 ms. Found 2 JPA repository interfaces.
2021-06-01 15:15:07.537  INFO 39132 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2021-06-01 15:15:07.539  INFO 39132 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2021-06-01 15:15:07.539  INFO 39132 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.45]
2021-06-01 15:15:07.576  INFO 39132 --- [main] o.a.c.c.C.[Tomcat-1].[localhost].[/]     : Initializing Spring embedded WebApplicationContext
2021-06-01 15:15:07.577  INFO 39132 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 718 ms
2021-06-01 15:15:07.669  INFO 39132 --- [main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2021-06-01 15:15:07.686  INFO 39132 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Starting...
2021-06-01 15:15:07.756  INFO 39132 --- [main] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Start completed.
2021-06-01 15:15:07.757  INFO 39132 --- [main] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.PostgreSQL81Dialect
2021-06-01 15:15:07.990  INFO 39132 --- [main] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2021-06-01 15:15:07.991  INFO 39132 --- [main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2021-06-01 15:15:08.541  WARN 39132 --- [main] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2021-06-01 15:15:08.842  INFO 39132 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 37615 (http) with context path ''
2021-06-01 15:15:09.159  INFO 39132 --- [main] o.a.maven.surefire.booter.ForkedBooter   : Started ForkedBooter in 2.343 seconds (JVM running for 14.79)
2021-06-01 15:15:09.206  INFO 39132 --- [main] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [com.josh.emailfunctionality.controller.EmailControllerTest]: no resource found for suffixes {-context.xml, Context.groovy}.
2021-06-01 15:15:09.228  INFO 39132 --- [main] .b.t.c.SpringBootTestContextBootstrapper : Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
2021-06-01 15:15:09.232  INFO 39132 --- [main] .b.t.c.SpringBootTestContextBootstrapper : Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@48720dcf, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@2f65c341, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@1e1908ab, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@7680cb52, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@2e44bc38, org.springframework.test.context.support.DirtiesContextTestExecutionListener@2fba0d7a, org.springframework.test.context.transaction.TransactionalTestExecutionListener@79b01476, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@99a8a49, org.springframework.test.context.event.EventPublishingTestExecutionListener@234523e, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@7d9a5edf, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@4f0e6be, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@59e5ecab, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@79e0f00b, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@5623354e, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@5565fc22]
2021-06-01 15:15:09.342  INFO 39132 --- [main] c.j.e.controller.EmailControllerTest     : Starting EmailControllerTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39132 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:15:09.348  INFO 39132 --- [main] c.j.e.controller.EmailControllerTest     : No active profile set, falling back to default profiles: default
2021-06-01 15:15:09.713  INFO 39132 --- [main] c.j.e.controller.EmailControllerTest     : Started EmailControllerTest in 0.471 seconds (JVM running for 15.345)
2021-06-01 15:15:09.728  INFO 39132 --- [main] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2021-06-01 15:15:09.728  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2021-06-01 15:15:09.730  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 1 ms
2021-06-01 15:15:09.815  INFO 39132 --- [main] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2021-06-01 15:15:09.816  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2021-06-01 15:15:09.819  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 0 ms
2021-06-01 15:15:09.882  INFO 39132 --- [main] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2021-06-01 15:15:09.884  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2021-06-01 15:15:09.887  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 2 ms
2021-06-01 15:15:09.895  WARN 39132 --- [main] o.s.web.servlet.PageNotFound             : No mapping for GET /api/v1/status
2021-06-01 15:15:09.908  INFO 39132 --- [main] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2021-06-01 15:15:09.909  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2021-06-01 15:15:09.913  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 1 ms
2021-06-01 15:15:09.917  WARN 39132 --- [main] o.s.web.servlet.PageNotFound             : No mapping for POST /api/v1/email
2021-06-01 15:15:09.925  INFO 39132 --- [main] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2021-06-01 15:15:09.927  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2021-06-01 15:15:09.931  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 0 ms
2021-06-01 15:15:09.959  WARN 39132 --- [main] o.s.web.servlet.PageNotFound             : No mapping for POST /api/v1/secondary
2021-06-01 15:15:09.973  INFO 39132 --- [main] o.s.b.t.m.w.SpringBootMockServletContext : Initializing Spring TestDispatcherServlet ''
2021-06-01 15:15:09.977  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Initializing Servlet ''
2021-06-01 15:15:09.980  INFO 39132 --- [main] o.s.t.web.servlet.TestDispatcherServlet  : Completed initialization in 1 ms
2021-06-01 15:15:10.142  INFO 39132 --- [SpringContextShutdownHook] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2021-06-01 15:15:10.159  INFO 39132 --- [SpringContextShutdownHook] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2021-06-01 15:15:10.167  INFO 39132 --- [SpringContextShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2021-06-01 15:15:10.176  INFO 39132 --- [SpringContextShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Shutdown initiated...
2021-06-01 15:15:10.204  INFO 39132 --- [SpringContextShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Shutdown completed.
2021-06-01 15:15:10.224  INFO 39132 --- [SpringContextShutdownHook] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2021-06-01 15:16:22.075  INFO 39319 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39319 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:16:22.087  INFO 39319 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:16:23.458  INFO 39319 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 1.901 seconds (JVM running for 3.428)
2021-06-01 15:16:23.649  INFO 39319 --- [smtp:127.0.0.1:3025<-/127.0.0.1:48717] com.icegreen.greenmail.smtp.SmtpManager  : Created user login test@gmail.com for address test@gmail.com with password test@gmail.com because it didn't exist before.
2021-06-01 15:16:23.674 ERROR 39319 --- [main] c.j.e.service.EmailSendServiceImpl       :  In exception of AuthenticationFailedException 
2021-06-01 15:16:23.681 ERROR 39319 --- [main] c.j.e.service.EmailSendServiceImpl       : In exception of SMTPSendFailedException daily limit failed for test@gmail.com
2021-06-01 15:16:23.694 ERROR 39319 --- [main] c.j.e.service.EmailSendServiceImpl       : In exception of MailConnectException
2021-06-01 15:16:23.699 ERROR 39319 --- [main] c.j.e.service.EmailSendServiceImpl       : In exception of SMTPAddressFailedException
2021-06-01 15:16:23.702 ERROR 39319 --- [main] c.j.e.service.EmailSendServiceImpl       : Other exception For final common exception
2021-06-01 15:20:53.938  INFO 39630 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39630 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:20:53.954  INFO 39630 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:20:55.645  INFO 39630 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 2.239 seconds (JVM running for 3.878)
2021-06-01 15:21:27.456  INFO 39677 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39677 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:21:27.467  INFO 39677 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:21:29.656  INFO 39677 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 3.307 seconds (JVM running for 5.692)
2021-06-01 15:24:12.962  INFO 39861 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39861 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:24:12.973  INFO 39861 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:24:14.552  INFO 39861 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 2.294 seconds (JVM running for 3.836)
2021-06-01 15:24:27.131  INFO 39901 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39901 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:24:27.136  INFO 39901 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:24:29.368  INFO 39901 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 3.272 seconds (JVM running for 5.388)
2021-06-01 15:24:51.106  INFO 39981 --- [main] c.j.e.service.EmailSendServiceImplTest   : Starting EmailSendServiceImplTest using Java 15.0.2 on hp-HP-ProBook-430-G4 with PID 39981 (started by hp in /home/hp/SpringHibernate/MyProject/Oncot-Email-Functionality-SpringBoot)
2021-06-01 15:24:51.123  INFO 39981 --- [main] c.j.e.service.EmailSendServiceImplTest   : No active profile set, falling back to default profiles: default
2021-06-01 15:24:53.843  INFO 39981 --- [main] c.j.e.service.EmailSendServiceImplTest   : Started EmailSendServiceImplTest in 3.79 seconds (JVM running for 6.457)
